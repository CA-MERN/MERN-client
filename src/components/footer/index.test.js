import { shallow } from "enzyme";
import React from "react";

import Footer from ".";

it("renders a snapshot", () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});
