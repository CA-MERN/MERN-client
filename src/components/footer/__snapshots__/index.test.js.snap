// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  class="footerBox"
>
  <div>
    Â© Adrienne Smith & Shelby El-rassi 2020
  </div>
</div>
`;
